src/lib.rs

```diff
@@ -61,7 +61,7 @@ extern crate unicase;
 
 mod scanners;
 mod entities;
-mod escape;
+pub mod escape;
```

Comment:
My understanding is this increases the public API surface? If so, worth mentioning in commit message.

Reply:
That is again an on oversight on my part. That was only public for benchmarking purposes. Will remove.